pkg/test/framework/components/pilot/kube.go

```diff
@@ -19,16 +19,12 @@ import (
 	"io"
 	"net"
 
-	multierror "github.com/hashicorp/go-multierror"
+	"istio.io/istio/pkg/test/framework/resource"
 
-	"istio.io/istio/pkg/test/framework/api/component"
-	"istio.io/istio/pkg/test/framework/api/components"
-	testContext "istio.io/istio/pkg/test/framework/api/context"
-	"istio.io/istio/pkg/test/framework/api/descriptors"
-	"istio.io/istio/pkg/test/framework/api/lifecycle"
-	"istio.io/istio/pkg/test/framework/runtime/api"
-	"istio.io/istio/pkg/test/framework/runtime/components/environment/kube"
+	"github.com/hashicorp/go-multierror"
```

Comment:
does this not cause linter issues? I seem to recall our recent changing of linters now required the alias here?

Reply:
It looks like not anymore. I think our linter rules just got way more relaxed. It doesn't even seem to care about ordering.

Reply:
We updated goimports. For the second time, they changed how they sort imports :( It's now not as strict.